nominal/cli/util/click_log_handler.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import annotations
+
+import logging
+import typing
+
+import click
+
+
+class ClickLogHandler(logging.StreamHandler[typing.TextIO]):
+    """Logging stream handler that routes and styles log messages through click
+    instead of directly routing to stderr.
+
+    This has several advantages, namely, differing log levels can be printed differently
+    to provide additional visual amplification of errors and warnings. Furthermore,
+    there are numerous improvements to support within windows.
+
+    See: https://click.palletsprojects.com/en/8.1.x/utils/#printing-to-stdout
+    """
+
+    LEVEL_TO_COLOR_MAP = {
+        logging.DEBUG: "blue",
+        logging.INFO: "cyan",
+        logging.WARNING: "yellow",
+        logging.ERROR: "red",
+        logging.FATAL: "bright_red",
```

Comment:
unimportant information: logging.WARN and logging.CRITICAL are equivalent but preferred over WARNING and FATAL respectively

Reply:
oops, I thought it was the other way around!